Gruntfile.js

```diff
@@ -332,32 +331,33 @@ module.exports = (grunt) => {
       .then((tests) => {
         grunt.log.writeln('Running click tests in parallel... (this will take a while...)');
         return Promise.all(
-          tests.map((file) => {
+          tests.map(async (file) => {
```

Comment:
The whole `clickParallel` task still has a bunch of `then`. Why is only this part rewritten?